server/plugin.go

```diff
@@ -138,17 +139,44 @@ func (p *Plugin) GetURL() string {
 	return *config.ServiceSettings.SiteURL + "/plugins/" + pluginID
 }
 
+func (p *Plugin) OnDisconnectedTokenHandler(userID string) {
+	p.API.LogDebug("OnDisconnectedTokenHandler", "userID", userID)
+	teamsUserID, err := p.store.MattermostToTeamsUserID(userID)
+	if err != nil {
+		p.API.LogWarn("Unable to get teams user id from mattermost to user", "userID", userID, "error", err.Error())
+		return
+	}
+	if err2 := p.store.SetUserInfo(userID, teamsUserID, nil); err2 != nil {
+		p.API.LogWarn("Unable clean invalid token for the user", "userID", userID, "error", err2.Error())
+		return
+	}
```

Comment:
Can we explicitly track this disconnection in a new column `DisconnectedAt`? This will allow us to enumerate the users that were disconnected and, for example, produce a report.